Formula/embulk.rb

```diff
@@ -8,12 +8,10 @@ class Embulk < Formula
 
   depends_on :java
```

Comment:
Should we maybe be a bit more specific about the supported Java version, e.g. `depends_on :java => "1.7+"`? (Don't know what the minimum requirement is.) On OS X, an unspecified version essentially means 1.6 or newer.